### PR TITLE
Add a helm chart for Chaos Engine Mattermost application

### PR DESCRIPTION
**Pull request by spirosoik**
#### Summary
Chaos Engine is a Mattermost Application which is used to create gamedayys tight coupled with
Mattermost as collaboration tool. Helm chart will help deploy in a kubernetes cluster.

#### Ticket Link
Ticket: https://mattermost.atlassian.net/browse/MM-38883

### CODE REVIEW

File: charts/mattermost-chaos-engine/values.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: mattermost/mattermost-app-chaosengine
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
```
Comment (stafot) [inline charts/mattermost-chaos-engine/values.yaml:11]: @spirosoik Ideally we don't want latest tag.

File: charts/mattermost-chaos-engine/values.yaml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+autoscaling:
```
Comment (stafot) [inline charts/mattermost-chaos-engine/values.yaml:79]: This block is not needed probably

Comment (spirosoik) [reply charts/mattermost-chaos-engine/values.yaml:79]: why not let's keep it

File: charts/mattermost-chaos-engine/values.yaml (hunk 1 above)
Comment (spirosoik) [reply charts/mattermost-chaos-engine/values.yaml:11]: as default value it's fine right? typically it shouls respect chart app version if you don't pass the tag at all

**Comment by spirosoik** [comment]
@stafot @stylianosrigas @angeloskyratzakos @Mshahidtaj  please have a look

**Comment by stafot** [comment]
@spirosoik Did you understand why circleCI fails?

**Review by stafot — APPROVED** [review]

**Comment by spirosoik** [comment]
> @spirosoik Did you understand why circleCI fails?

yes was about test connection and 404 of wget when chaos engine run. So I used a URL which always exist to get 200 and it's representative. Check my last commit

**Review by Mshahidtaj — APPROVED** [review]
LGTM

**Pull request merged**